parsec/core/pki/submitter.py

```diff
@@ -236,7 +236,7 @@ async def poll(
             PkiEnrollmentInfoError
             PkiEnrollmentInfoNotFoundError
         """
-        #  Tuple[PkiEnrollmentStatus, DateTime, Optional[LocalDevice]]:
+        #  Tuple[PkiEnrollmentStatus, DateTime, LocalDevice | None]:
```

Comment:
Should we remove this comment?

Reply:
I kept it in case it's useful, but we could definitely remove it